### PR TITLE
[RDY] School mapgen JSONify

### PR DESCRIPTION
**Pull request by ZhilkinSerg**
Conversion of `school` mapgen from `map::draw_map` to JSON.

### CODE REVIEW

File: data/json/vehicle_groups.json [hunk 1]
```diff
@@ -229,5 +229,14 @@
             [ "handjack", 150 ],
             [ "wheelbarrow", 100 ]
         ]
+    },
+    {
+        "id": "school_vehicles",
+        "type": "vehicle_group",
+        "//": "School bus or fire truck found near school locations",
```
Comment (Night-Pryanik) [inline data/json/vehicle_groups.json]: Fire truck at school?

Comment (ZhilkinSerg) [inline data/json/vehicle_groups.json]: That was not my idea - see https://github.com/CleverRaven/Cataclysm-DDA/blob/master/src/mapgen.cpp#L5829.

Comment (kevingranade) [inline data/json/vehicle_groups.json]: It's an emergency, school seems like a high priority place to dispatch EMS.

**Comment by Coolthulhu** [comment]
I don't know how to fix the migration error and the error corrupts saves, so it is reverted for now.
Re-PR it once you get the bug fixed.

**Comment by ZhilkinSerg** [comment]
@Coolthulhu, once I get the fix, should I put all the changes (including changes from original commit) to new pull request or only savegame fix?

**Comment by Coolthulhu** [comment]
All

**Pull request merged**